docker/conf/certificates/MinervaRootCA-2039.cer

```diff
@@ -0,0 +1,38 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I do not think that it is good idea to add certificate to public repo. We should provide opportunity to mount it, instead

Reply:
I raised the same question. As I understand Erlend approved this. Is that correct @dangernil ?

Reply:
I have added in so that they are copied from image instead, we just need to update latest image of minio to include those certs. Will update when that´s ok